app/sections/video-banner/index.tsx

```diff
@@ -96,7 +96,7 @@ function getPlayerSize(id: string) {
       let rect = section.getBoundingClientRect();
       let aspectRatio = rect.width / rect.height;
       if (aspectRatio < 16 / 9) {
-        return {width: 'auto', height: '100%'};
+        return {width: 'auto', height: '100vh'};
```

Comment:
Please check this carefully by resizing the browser from wide to small to check, update the section height config (from small to full screen) and double check again.

Reply:
I just ran the test with nr dev and nr preview and tested again on the screens and didn't see any problem. You can build to production to see if there are any errors.